include/plugin_manager/rest_framework/callback_handler.hpp

```diff
@@ -0,0 +1,35 @@
+/*
+ * Punica - LwM2M server with REST API
+ * Copyright (C) 2019 8devices
+ *
+ * This program is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License,
+ * or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <https://www.gnu.org/licenses/>.
+ *
+ */
+
+#ifndef CALLBACK_HANDLER_HPP
+#define CALLBACK_HANDLER_HPP
+
+typedef int (*ulfius_callback_function_t)(const struct _u_request *, struct _u_response *, void *);
```

Comment:
Is this used anywhere? Also is abstract interfaces are created, ulfius types should not be included.